### ISSUE TITLE
[Markup and meta data] Visual labels aren't included in the accessible label. (09.05.2)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29344]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Trevor Pierce (Accessibility)

---

### Platform Issue
Visual labels aren't included in the accessible label.

### Issue Details
Remove buttons do not announce what is being removed to screen readers.

### Link, screenshot or steps to recreate
![Screen Shot 2021-08-26 at 5 11 04 PM](https://user-images.githubusercontent.com/934879/131042870-5d051634-6af7-4787-b294-3e91e519df67.png)

### VA.gov Experience Standard
[Cateogy Number 09, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.5.3_A

---

### Platform Recommendation
Remove buttons in the Compare Institutions tray and Comparison page should always have an aria-label that announces what school is being removed. So screen readers would hear "Remove Florida State, button".

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/18561
